votrfront/js/PrehladStudiaPage.js

```diff
@@ -69,10 +69,31 @@ export function PrehladStudiaObdobia() {
   });
 }
 
+// funguje len pre 5 rocne studium !!!
+// ked bude mat bc studium dlzku inu ako 3, alebo magisterske inu ako 2, bugne
+function GetOficialnaDlzkaStudia(studium){
+
+  // asi najlepsi test
+  if (studium.sp_doplnujuce_udaje.includes("bakalársky")) return 3;
+  if (studium.sp_doplnujuce_udaje.includes("magisterský")) return 2;
```

Comment:
Tu by som bol zvedavy, ci to bude fungovat aj pre Farmaceutov, ktori maju 5 rocne studium, popripade medikov s 6 rokmi.